Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern_test.py

```diff
@@ -48,8 +40,39 @@ def test_extract_existing_mitre_id(mocker):
     - run the ExtractAttackPattern script
     Validate that name extracted successfully from the ID.
     """
-    mocker.patch.object(demisto, 'executeCommand', return_value=[{}, {}, {'Contents':
-                        [{'id': 'T1530', 'value': 'Data from Cloud Storage Object'}]}])
+    mocker.patch.object(eap, 'get_mitre_results', return_value=[{
+        'id': 'T1530', 'value': 'Data from Cloud Storage Object'}])
 
     indicators = is_valid_attack_pattern(['T1530'])
     assert indicators == ['Data from Cloud Storage Object']
+
+
+def test_extract_with_value_error(mocker):
```

Comment:
please add doc string to explain the test

Reply:
done